worlds/terraria/docs/setup_en.md

```diff
@@ -3,20 +3,32 @@
 ## Required Software
 
 Download and install [Terraria](https://store.steampowered.com/app/105600/Terraria/)
-and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Steam
+and [tModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Steam
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open tModLoader
+3. Go to "Workshop" -> "Manage Mods" and enable the Archipelago mod
```

Comment:
This should be emphasized with bold text:
3. Go to **Workshop -> Manage Mods** and enable the Archipelago mod.